demos/bridge/primefaces3-portlet/src/main/webapp/views/applicant.xhtml

```diff
@@ -66,7 +66,7 @@
 									<f:selectItem itemLabel="#{i18n['select']}" itemValue="" />
 									<f:selectItems itemValue="#{province.provinceId}" itemLabel="#{province.provinceName}"
 										value="#{listModelBean.provinces}" var="province" />
-									<p:ajax update="provinceIdField" />
+									<f:ajax render="provinceIdField" />
 								</h:selectOneMenu>
```

Comment:
That's OK -- probably better to use p:selectOneMenu and then create a new ticket for the test failures, so that we can fix the test. It probably fails due to some change in the DOM.